src/menu.ts

```diff
@@ -1,13 +1,15 @@
 import { updateAndDraw as pongUpdateAndDraw } from "./pong";
 import { updateAndDraw as bulletHellUpdateAndDraw } from "./bullet-hell";
+import { updateAndDraw as setUpdateAndDraw } from "./set";
 
-const games = {
-  pong: pongUpdateAndDraw,
-  "bullet hell": bulletHellUpdateAndDraw,
-};
-const gameNames = Object.keys(games);
+const games = [
```

Comment:
curious to get your take on using tuples instead of an object. it was the easiest way I could think of to fix the type error on line 95